src/python/bin/aqueduct

```diff
@@ -57,6 +57,77 @@ def update_executable_permissions():
+            return False
+        else:
+            return True
+
+def update_ui_version():
+    print("Updating UI version to %s" % package_version)
+    try:
+        execute_command(["rm", "-rf", ui_directory])
+        os.mkdir(ui_directory)
+        generate_version_file(os.path.join(ui_directory, "__version__"))
+        s3_prefix = "https://aqueduct-ai.s3.us-east-2.amazonaws.com/assets/%s/ui" % package_version
+        execute_command(["curl", os.path.join(s3_prefix, "ui.zip"), "--output", os.path.join(ui_directory, "ui.zip")])
+        execute_command(["unzip", os.path.join(ui_directory, "ui.zip"), "-d", ui_directory])
+        execute_command(["rm", os.path.join(ui_directory, "ui.zip")])
+        execute_command(["npm", "install"], os.path.join(ui_directory, "app"))
+        execute_command(["npm", "run", "build"], os.path.join(ui_directory, "app"))
+    except Exception as e:
+        print(e)
+        execute_command(["rm", "-rf", ui_directory])
+        exit(1)
+
+def download_golang_binaries(s3_prefix):
+    system = platform.system()
+    arch = platform.machine()
+    if system == "Linux" and arch == "x86_64":
+        print("Operating system is Linux with architecture amd64.")
+        execute_command(["curl", os.path.join(s3_prefix, "bin/linux_amd64/server"), "--output", os.path.join(server_directory, "bin/server")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/linux_amd64/executor"), "--output", os.path.join(server_directory, "bin/executor")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/linux_amd64/migrator"), "--output", os.path.join(server_directory, "bin/migrator")])
+    elif system == "Darwin" and arch == "x86_64":
+        print("Operating system is Mac with architecture amd64.")
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_amd64/server"), "--output", os.path.join(server_directory, "bin/server")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_amd64/executor"), "--output", os.path.join(server_directory, "bin/executor")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_amd64/migrator"), "--output", os.path.join(server_directory, "bin/migrator")])
+    elif system == "Darwin" and arch == "arm64":
+        print("Operating system is Mac with architecture arm64.")
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_arm64/server"), "--output", os.path.join(server_directory, "bin/server")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_arm64/executor"), "--output", os.path.join(server_directory, "bin/executor")])
+        execute_command(["curl", os.path.join(s3_prefix, "bin/darwin_arm64/migrator"), "--output", os.path.join(server_directory, "bin/migrator")])
+    else:
+        raise Exception("Unsupported operating system and architecture combination: %s, %s" % (system, arch))
+
+def update_server_version():
+    print("Updating server version to %s" % package_version)
```

Comment:
I don't see where `package_version` is ever updated from 0.0.1 - am I missing something?

Reply:
When we decide to cut a release, we will make a PR that updates this hardcoded version number from 0.0.1 to 0.0.2.